packages/nextjs/components/Footer.tsx

```diff
@@ -12,16 +12,18 @@ export default function Footer() {
 
   return (
     <div className="min-h-0 p-5 flex justify-between items-center flex-col sm:flex-row gap-4">
-      <div>
-        <div className="fixed m-4 bottom-0 left-0">
-          {ethPrice > 0 && (
-            <div className="btn btn-primary btn-sm font-normal cursor-auto">
-              <CurrencyDollarIcon className="h-4 w-4 mr-0.5" />
-              <span>{ethPrice}</span>
-            </div>
-          )}
+      <div className="flex w-full sm:w-auto items-center justify-between mx-4">
+        {ethPrice > 0 && (
+          <div className="btn btn-primary btn-sm font-normal cursor-auto">
+            <CurrencyDollarIcon className="h-4 w-4 mr-0.5" />
+            <span>{ethPrice}</span>
+          </div>
+        )}
+        <div className="sm:hidden text-sm">
+          <SwitchTheme key="xs" />
```

Comment:
You removed the "fixed" behavior from the footer. Check in main. We want the eth price & the switch to follow you when your scroll.

You can removed this from this PR and open an issue to make the address the issues with the footer on mobile.

https://user-images.githubusercontent.com/2486142/216039550-bbcd7e77-027e-491e-8e2d-7c29ae9626d8.mp4

Reply:
I did it intentionally since fixed elements look bad on mobiles and are placed in the footer component.

https://user-images.githubusercontent.com/25638585/216044356-3babe022-5271-4b69-81b8-a5637e11cdf3.mov

I'll revert that changes and create an issue then.

Reply:
> I did it intentionally since fixed elements look bad on mobiles and are placed in the footer component.



We might not want the elements to be fixed on mobile.  But you broke it on desktop.

Reply:
👌  Reverted changes, created https://github.com/scaffold-eth/se-2/issues/123